<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NormalizeInput</class>
 <widget class="QWidget" name="NormalizeInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>274</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MyQtEnumComboBox" name="MyQtEnumComboBox_fieldsTypes">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>185</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The current time</string>
   </property>
   <property name="whatsThis">
    <string>Enum combo box time.</string>
   </property>
  </widget>
  <widget class="MyQtEnumComboBox" name="MyQtEnumComboBox_range">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>156</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The current time</string>
   </property>
   <property name="whatsThis">
    <string>Enum combo box time.</string>
   </property>
  </widget>
  <widget class="MyQtEnumComboBox" name="MyQtEnumComboBox_normalizeMethod">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>127</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The current time</string>
   </property>
   <property name="whatsThis">
    <string>Enum combo box time.</string>
   </property>
  </widget>
  <widget class="MyQtEnumComboBox" name="MyQtEnumComboBox_fieldRoll">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>98</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The current time</string>
   </property>
   <property name="whatsThis">
    <string>The analog clock widget displays the current time.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_selected">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>247</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changeOrder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Values Order</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_max">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_min">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyQtEnumComboBox</class>
   <extends>QComboBox</extends>
   <header>MyQtEnumComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
